test/test_schedules/test_status_initial_off.cpp

```diff
@@ -31,32 +30,40 @@ void test_status_initial_off_inj4(void)
 #if INJ_CHANNELS >= 5
 void test_status_initial_off_inj5(void)
 {
+#if ING_CHANNELS >= 5
```

Comment:
There's a typo in `ING_CHANNELS` here. I don't believe that `#if` is required anyway as there's already one around the coverall function declaration. 

Same applies for all of these >=5

Reply:
Fixed - and addressed a tennsy unit test build issue.